core/src/main/java/io/questdb/griffin/ExpressionParser.java

```diff
@@ -287,6 +288,45 @@ void parseExpr(GenericLexer lexer, ExpressionParserListener listener) throws Sql
 
                         break;
 
+                    case 'g':
+                    case 'G':
+                        if (SqlKeywords.isGeoHashKeyword(tok)) {
+                            CharSequence originalGeoHashKeyword = GenericLexer.immutableOf(tok);
+                            CharSequence peek = lexer.peek();
+                            if (peek == null || peek.charAt(0) != '(') {
+                                processDefaultBranch = true;
+                                break;
+                            }
+                            SqlUtil.fetchNext(lexer); // consume '('
+                            tok = SqlUtil.fetchNext(lexer);
+                            if (tok != null && tok.charAt(0) != ')') {
+                                SqlParser.parseGeoHashSize(lexer.lastTokenPosition(), tok);
```

Comment:
allocating code

Reply:
line 303? apologies, I do not understand what action should be taken, the '(' needs to be consumed, the tok fetched and checked for correctness?, or should it be not checked at this point?